templates/templated_email/source/staff_confirm_order.mjml

```diff
@@ -0,0 +1,56 @@
+<mjml>
+  <mj-head>
+    <mj-include path="./shared/styles.mjml" />
+  </mj-head>
+  <mj-body>
+    <mj-raw>
+      {% load i18n %}
+      {% load i18n_address_tags %}
+    </mj-raw>
+    <mj-include path="./shared/header.mjml" />
+    <mj-section>
+      <mj-column>
+        <mj-text font-size="16px">
+          {% trans "Hi!" context "Standard e-mail greeting" %}
+        </mj-text>
+        <mj-text>
+          {% blocktrans trimmed context "Order confirmation staff e-mail text" %}
+            Below is the list of ordered products. To see payment details please visit: <a href="{{ order_details_url }}">{{ order_details_url }}</a>
```

Comment:
I'd slightly change the body here:
"Someone placed a new order in your store. To see payment details [...]".